tests/torch/fx/test_smooth_quant.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Callable, Dict, Type
+
+import numpy as np
+import openvino.runtime as ov
+import pytest
+import torch
+from torch._export import capture_pre_autograd_graph
```

Comment:
```
W0814 17:55:55.410000 125432503427712 torch/_export/__init__.py:96] |     !!!   WARNING   !!!    |
W0814 17:55:55.410000 125432503427712 torch/_export/__init__.py:97] +============================+
W0814 17:55:55.410000 125432503427712 torch/_export/__init__.py:98] capture_pre_autograd_graph() is deprecated and doesn't provide any function guarantee moving forward.
W0814 17:55:55.410000 125432503427712 torch/_export/__init__.py:99] Please switch to use torch.export instead.
```

Reply:
I know about this issue, but
* I did not find the way to use torch.export instead, it produces [torch.export.ExportedProgram](https://pytorch.org/docs/stable/export.html#overview), but our algorithm is based on `torch.fx.GraphModule` (as well as the [official documentation on torch quantization ](https://pytorch.org/tutorials/prototype/pt2e_quant_x86_inductor.html#capture-fx-graph)
* I suggest to implement to finish all algorithms using `capture_pre_autograd_graph` and then hopefully migrate to `torch.export` once for all